src/components/theme/updateBodyClassName.ts

```diff
@@ -5,6 +5,8 @@ const b = block('root');
 const rootClassName = b();
 const [, lightModifier] = b({theme: 'light'}).split(/\s+/);
 const [, darkModifier] = b({theme: 'dark'}).split(/\s+/);
+const [, lightHcModifier] = b({theme: 'light-hc'}).split(/\s+/);
+const [, darkHcModifier] = b({theme: 'dark-hc'}).split(/\s+/);
```

Comment:
I think we can add some helper here, like

```suggestion
function getModifier(className: string): string {
  const [, modifier] = className.split(/\s+/)

  return modifier;
}

const lightModifier = getModifier(b({theme: 'light'}));
const darkModifier = getModifier(b({theme: 'dark'}));
const lightHcModifier = getModifier(b({theme: 'light-hc'}));
const darkHcModifier = getModifier(b({theme: 'dark-hc'}));
```